### ISSUE TITLE
Parsing error with output of 'df'

### ISSUE THREAD

**Issue by Grothesk242** [iobroker-community-adapters/ioBroker.rpi2#158]
It seems that evaluation of ``df`` outout goes wrong recently:

```
2023-10-13 22:37:23.826  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12545616  97873460  12% /
/1024
2023-10-13 22:37:23.833  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12545616  97873460  12% /
/1024
2023-10-13 22:38:23.721  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12545976  97873100  12% /
/1024
2023-10-13 22:38:23.727  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12545976  97873100  12% /
/1024
2023-10-13 22:39:23.686  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12546160  97872916  12% /
/1024
2023-10-13 22:39:23.690  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12546160  97872916  12% /
/1024
2023-10-13 22:40:23.707  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12546464  97872612  12% /
/1024
2023-10-13 22:40:23.709  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12546464  97872612  12% /
/1024
2023-10-13 22:41:06.907  - error: parcel.0 (154799) Error: read ETIMEDOUT
2023-10-13 22:41:09.830  - info: parcel.0 (154799) Login to Amazon successful
2023-10-13 22:41:23.730  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12546776  97872300  12% /
/1024
2023-10-13 22:41:23.733  - error: rpi2.0 (159251) Cannot evaluate: Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root      115129216 12546776  97872300  12% /
/1024
```

**Comment by Grothesk242** [comment]
``````

**Comment by VitBeck** [comment]
have the same issue and no idea how to solve it...

**Comment by VitBeck** [comment]
rpi2.0, v1.3.2

2023-11-19 20:50:43.198 | error | Cannot evaluate: Filesystem 1K-blocks Used Available Use% Mounted on /dev/sda2 245607812 14466796 220567108 7% / /1024
-- | -- | --

```
pi@iobroker:~ $ df
Filesystem     1K-blocks     Used Available Use% Mounted on
udev             3726200        0   3726200   0% /dev
tmpfs             799764     1316    798448   1% /run
/dev/sda2      245607812 14467228 220566676   7% /
tmpfs            3998812      148   3998664   1% /dev/shm
tmpfs               5120       16      5104   1% /run/lock
/dev/sda1         522232    75070    447162  15% /boot/firmware
tmpfs             799760       36    799724   1% /run/user/1000
```

**Comment by Apollon77** [comment]
In fact this adapter i sexecuting linux commands ... defined in https://github.com/iobroker-community-adapters/ioBroker.rpi2/blob/master/io-package.json#L235-L366 ... so maybe check which command it is and try to find the correct one for rpi4 and we know what to change. Any help appreciated ....

**Comment by Grothesk242** [comment]
> so maybe check which command it is

Its ``df``

**Comment by Grothesk242** [comment]
Fixed in upcoming release 2.0. Code needs to be tested though.

**Comment by Grothesk242** [comment]
Fixed
